Foundation/NSJSONSerialization.swift

```diff
@@ -125,8 +125,8 @@ public class JSONSerialization : NSObject {
        The data must be in one of the 5 supported encodings listed in the JSON specification: UTF-8, UTF-16LE, UTF-16BE, UTF-32LE, UTF-32BE. The data may or may not have a BOM. The most efficient encoding to use for parsing is UTF-8, so if you have a choice in encoding the data passed to this method, use UTF-8.
      */
     /// - Experiment: Note that the return type of this function is different than on Darwin Foundation (Any instead of AnyObject). This is likely to change once we have a more complete story for bridging in place.
-    public class func jsonObject(with data: Data, options opt: ReadingOptions = []) throws -> Any {
```

Comment:
Implementation NSJSONSerialization.jsonObject(with: stream) internally needs to call the api NSJSONSerialization.jsonObject(with: data). Currently, return type of NSJSONSerialization.jsonObject(with: data) returns 'Any' instead of 'AnyObject' and it is not possible to cast 'Any' to 'AnyObject'. 

Wrote a query to the mailing list regarding the above hiccup. Following is the response from Philippe Hausler:
"We had changed the return type of JSONSerialization to be Any such that it avoided the caller needing to bridge types (which works poorly on linux). Even though this is an API change technically, just change the return type to match the other JSONSerialization methods."

@phausler: In accordance with the above suggestion, modified the return type and the related code (including the existing test cases) to resolve errors caused by the change.